tests/ui/impl-trait/in-ctfe/match-arm-exhaustive.rs

```diff
@@ -0,0 +1,24 @@
+//! Check that pattern matching can observe the hidden type of opaqaue types.
```

Comment:
(Tiny typo happening twice)

```suggestion
//! Check that pattern matching can observe the hidden type of opaque types.
```